cmd/utils/flags.go

```diff
@@ -900,10 +900,25 @@ var (
 	}
 	UnsafeDebugDisableFlag = &cli.BoolFlag{
 		Name:    "rpc.unsafe-debug.disable",
-		Usage:   "Disable unsafe debug APIs (traceTransaction, writeXXX, ...).",
+		Usage:   "Disable unsafe debug APIs (traceTransaction, traceChain, ...).",
 		Aliases: []string{"http-rpc.unsafe-debug.disable"},
 		EnvVars: []string{"KLAYTN_RPC_UNSAFE_DEBUG_DISABLE"},
 	}
+	// TODO-klaytn: Consider limiting the non-debug heavy apis.
+	HeavyDebugRequestLimitFlag = &cli.IntFlag{
+		Name:    "rpc.unsafe-debug.heavy-debug.requestlimit",
```

Comment:
Flag name should match env vars:
```suggestion
		Name:    "rpc.unsafe-debug.heavy-debug.request-limit",
```